bot/__main__.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+import inspect
+import os
+import pkgutil
+
+from dotenv import load_dotenv
+
+from bot import exts
+from bot.bot import bot
+
+if __name__ == "__main__":
```

Comment:
Truthfully, I think this is a bit too "cute" and magical. It would be better™ to just import the packages and then call `.setup()` on them, or have a "extensions" array (and then iterate over them and call `.setup()` on them.

Reply:
I purposely went with a more "magical" method because I didn't want to have an extensions array we'd have to adjust and change with each extension added. Do you think it'd be better to have call out the different extensions and load them individually? @travigd

Reply:
It's prooooooobably fine. Java land does this a bunch with their auto-wired beans.

That being said, I do think it's **better** to just have an array where you specify all of them explicitly.